packages/file_selector/file_selector_ios/ios/Classes/file_selector_ios-umbrella.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Foundation/Foundation.h>
+#import <file_selector_ios/FFSFileSelectorPlugin.h>
```

Comment:
Missing newline (@stuartmorgan we don't have a linter for this?)

Reply:
Fixed. Apparently clang-format doesn't enforce the header blocking even with Google style.